File: 1000-1200 Python/ATTENDU.py
T = int(input())

for i in range(T):
    N, attend = int(input()), input()
    absent = 0

    for j in attend:
        if j == "0":
            absent += 1

    attendance = ((120-absent)/120) * 100

    if attendance >= 75:
        print("YES")
    else:
        print("NO")

File: 1000-1200 C/ATTENDU.c
#include <stdio.h>
int main(){
    int T, N;
    float attendance;     
    scanf("%d", &T);
    for (int i = 0; i < T; i++){
        scanf("%d", &N);
        char attend[N];
        float absent = 0;
        scanf("%s", &attend);

        for (int j = 0; j < N; j++){
            if (attend[j] == '0'){
                absent += 1;
            }
        }

        attendance = (1 - absent/120) * 100;
        (attendance >= 75) ? printf("YES\n") : printf("NO\n");    
    }
    return 0;
}
